authapi/views.py

```diff
@@ -1,16 +1,54 @@
 from django.contrib.auth.models import User
 from rest_framework import viewsets, status
+from rest_framework.generics import get_object_or_404
 from rest_framework.response import Response
 
 from authapi.models import SeedOrganization, SeedTeam
 from authapi.serializers import (
-    OrganizationSerializer, TeamSerializer, UserSerializer)
+    OrganizationSerializer, TeamSerializer, UserSerializer,
+    OrganizationUserSerializer)
 
 
 class OrganizationViewSet(viewsets.ModelViewSet):
     queryset = SeedOrganization.objects.all()
     serializer_class = OrganizationSerializer
 
+    def get_queryset(self):
+        '''We want to still be able to modify archived organizations, but they
+        shouldn't show up on list views.'''
+        if self.action == 'list':
+            return SeedOrganization.objects.filter(archived=False)
+        return self.queryset
```

Comment:
Related -- wondering if we need something similar for active and inactive users.